builtin/logical/aws/rotation.go

```diff
@@ -37,13 +38,13 @@ func (b *backend) rotateExpiredStaticCreds(ctx context.Context, req *logical.Req
 }
 
 // rotateCredential pops an element from the priority queue, and if it is expired, rotate and re-push.
-// If a cred was rotated, it returns true, otherwise false.
+// If a cred was ready for rotation, return true, otherwise return false.
 func (b *backend) rotateCredential(ctx context.Context, storage logical.Storage) (rotated bool, err error) {
```

Comment:
nit: wondering if we can also update the name of the returned parameter to distinguish that it is now 'ready for rotation' instead of something already being rotated
```suggestion
func (b *backend) rotateCredential(ctx context.Context, storage logical.Storage) (isReadyForRotation bool, err error) {
```